src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -218,6 +218,7 @@ def load_command_table(self, _):
         g.command('add-value', 'create_or_update_value')
         g.command('remove-value', 'delete_value')
 
+    # az group deployment
     with self.command_group('group deployment', resource_deployment_sdk) as g:
```

Comment:
>command_group [](start = 14, length = 13)

will group deployment, group deployment operation be deprecated, if so, you may need to add redirect here

Reply:
they will be deprecated. we'll add the deprecation info later in a separate PR.